IndomitableIbises/Bangazon-API-Sprint-1
# Department model, view, and url done.  Can post, put, delete

Link to Ticket
Closes #7 

Description of Proposed Changes
Added the Department model, view and URL

Steps to Test
```
git fetch --all
git checkout JCdepartmentresourcemodel
```

1. Delete existing database
2. python manage.py migrate
3. code .
4. Look at the code and see if you have questions
5. python manage.py runserver
6. You should see three clickable urls on our page. Click on "http://localhost:8000/departments/" and make sure you are able to add the name of the department.

Impacted Areas in Application

Need to add employees so we can uncomment the employee foreignkey on this one

*Models, views, and urls are impacted by this code

Definition of Done
The project must be fully documented. This includes the following:
Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
Every class must be documented with purpose, author, and methods.
Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
The project must be able to run fully, and without errors, on each teammate's system.
Fulfills every requirement.
Every line of code has been peer reviewed.
For projects that require unit testing, core functionality must be identified and have at least one test for each
